src/range_statement.rs

```diff
@@ -29,7 +29,7 @@ pub struct RangeStatement<P: Compressable> {
     pub seed_nonce: Option<Scalar>,
 }
 
-impl<P: Compressable + FromUniformBytes + Clone> RangeStatement<P> {
+impl<P: Copy + Compressable + FromUniformBytes + Clone + Precomputable> RangeStatement<P> {
```

Comment:
Same reason as above, dont like the copy here

Reply:
Removed in a new update.